app/modules/inventory/components/item_row.svelte

```diff
@@ -33,16 +35,38 @@
 <div class="item-row">
   <slot name="checkbox" />
 
-  <a href="{pathname}" on:click|stopPropagation={loadInternalLink}>
-    <div class="image-wrapper">
-      {#if image}<img src="{imgSrc(image, 128)}" alt="">{/if}
-    </div>
-    <div class="info">
-      <p class="title">{title}</p>
-      <p class="authors">{authors || ''}</p>
-    </div>
-    <p class="details">{details}</p>
-  </a>
+  <div class="middle">
+    <a
+      href="{pathname}"
+      on:click|stopPropagation={loadInternalLink}
+      class="show-item"
```

Comment:
when no author, cover is overflown since no minimum height is set on `.show-item`

here is my dev local display, you might find other details than the one mentioned above 

![2022-10-22_14-25](https://user-images.githubusercontent.com/5363918/197338907-1d054ca4-2d7b-4c7a-97ed-60c64447e60f.png)

Reply:
done in f14750f9e